src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -54,6 +40,27 @@
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WritePackageVersionsProps" />
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WritePackageUsageData" />
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WriteUsageReports" />
+  
+  <!-- Wraps the transitive repo references with additional metadata -->
+  <Target Name="GetRepositoryReferenceInfo"
+          DependsOnTargets="GetTransitiveRepositoryReferences">
+    <!-- SBRP is explicitly excluded since it is output to its own special directory, $(ReferencePackagesDir). -->
+    <ItemGroup Condition="'@(TransitiveRepositoryReference)' != ''">
+      <RepositoryReferenceInfo Include="@(TransitiveRepositoryReference)"
+                               Condition="'%(Identity)' != 'source-build-reference-packages'">
```

Comment:
Why not use `Exclude="source-build-reference-packages"` here instead? That has better evaluation perf is more readable.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/bdaa440797890f0ef606393025d6967d656c4917